core/src/main/java/org/sonarsource/sonarlint/core/container/storage/StorageContainerHandler.java

```diff
@@ -55,7 +57,7 @@
 
   public StorageContainerHandler(StorageAnalyzer storageAnalyzer, StorageRuleDetailsReader storageRuleDetailsReader, GlobalUpdateStatusReader globalUpdateStatusReader,
     PluginRepository pluginRepository, ModuleStorageStatusReader moduleStorageStatusReader, IssueStoreReader issueStoreReader, AllModulesReader allModulesReader,
-    StoragePaths storagePaths, StorageReader storageReader, TempFolder tempFolder, StorageFileExclusions storageExclusions) {
+    StoragePaths storagePaths, StorageReader storageReader, TempFolder tempFolder, StorageFileExclusions storageExclusions, ComponentContainer container) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "container". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)